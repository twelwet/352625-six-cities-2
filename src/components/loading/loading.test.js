import React from "react";
import renderer from "react-test-renderer";
import Loading from "./loading.jsx";

it(`Loading correctly renders after relaunch`, () => {

  const tree = renderer
    .create(<Loading />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
