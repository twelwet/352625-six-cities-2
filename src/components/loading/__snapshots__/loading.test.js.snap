// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading correctly renders after relaunch 1`] = `
<div
  style={
    Object {
      "left": "50%",
      "position": "absolute",
      "top": "50%",
    }
  }
>
  Loading...
</div>
`;
